### ISSUE TITLE
BERT Embeddings trainable

### ISSUE THREAD

**Issue by simonefrancia** [google-research/bert#226]
Hi,
in these days I was able to apply finetuning to my own dataset starting from BERT embeddings and calculating them on the fly, as you advise in (https://github.com/google-research/bert/issues/91).

First results are not not good because I get a constant 42% on validation set. My doubt is:
are BERT input embedding "trainable"? if not, how can i do that?

thanks

**Comment by anpark** [comment]
all layers are trainable
